### DISCUSSION TITLE
git push  error

### DISCUSSION THREAD

**Discussion by iamhrhp** [fineanmol/hacktoberfest#1733] [General]
remote: Permission to fineanmol/hacktoberfest.git denied to iamhrhp.
fatal: unable to access 'https://github.com/fineanmol/hacktoberfest.git/': The requested URL returned error: 403

getting this error while pushing the code i have tried to use ssh link to push the code bt same permission is denied

**Comment by KanishkRastogi46** [comment]
> remote: Permission to fineanmol/hacktoberfest.git denied to iamhrhp. fatal: unable to access 'https://github.com/fineanmol/hacktoberfest.git/': The requested URL returned error: 403
> 
> getting this error while pushing the code i have tried to use ssh link to push the code bt same permission is denied

I am also getting same error
